src/component/container/element_wrapper.tsx

```diff
@@ -46,7 +46,9 @@ export class ElementWrapper extends React.Component<ElementWrapperProps, Element
 		this.setState({
 			highlight: true
 		});
-		this.props.handleDragStart && this.props.handleDragStart(e);
+		if (typeof this.props.handleDragStart === 'function') {
```

Comment:
Why the exact check on function?
With the typings it is made sure that there can only be a `function` or `undefined` in `handleDragStart`

Reply:
Sure, but:

1. This makes the needed type clearer when reading
2. tslint did not like the convultion into one expression